### PR TITLE
Added jdbc + other things

### PR DESCRIPTION
**Pull request by oolshov**
- [ ] @dkozinets

Added jdbc + mysql + other things

### CODE REVIEW

File: src/main/java/com/github/orest/car_shop/Main.java [hunk 1]
```diff
@@ -1,45 +1,15 @@
...
-                System.out.println(CarShopService.getMessage("main"));
-                text = input.nextLine();
-            } else {
-                CarShopService.getBrand(text, allCars);
-                System.out.println(CarShopService.getMessage("main"));
-                text = input.nextLine();
-            }
-        }
+        SpringApplication.run(Main.class, args);
+        CarService.getAllCars();
```
Comment (oolshov) [inline src/main/java/com/github/orest/car_shop/Main.java]: потім заберу цю стрічку, забув)

File: src/main/java/com/github/orest/car_shop/service/DBWorkerService.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import lombok.Data;
+import java.sql.*;
+
+@AllArgsConstructor
+@Data
+public class DBWorkerService {
+    private static final String driverName = "com.mysql.jdbc.Driver";
+    private static final String url = "jdbc:mysql://localhost:3306/car_shop";
+    private static final String login = "root";
+    private static final String password = "root";
```
Comment (dkozinets) [inline src/main/java/com/github/orest/car_shop/service/DBWorkerService.java]: Put these variables into property file

File: src/main/java/com/github/orest/car_shop/service/DBWorkerService.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    private Statement statement;
+
+    public DBWorkerService() {
+        try {
+            Connection connection = DriverManager.getConnection(url, login, password);
```
Comment (dkozinets) [inline src/main/java/com/github/orest/car_shop/service/DBWorkerService.java]: You should close connection and statement. You can use try-with-resources or finally block
https://stackoverflow.com/questions/25864235/why-we-should-close-the-connection-in-jdbc-if-we-dont-do-it-what-will-happen

File: src/main/java/com/github/orest/car_shop/service/CarService.java [hunk 4]
```diff
@@ -63,12 +64,17 @@ public static void validateCars(List<Car> allCars) {
...
+    // this method return all available cars from DB
+    public static List<Car> getAllCars() throws SQLException {
 
-        validateCars(allCars);
+        ResultSet rs = new DBWorkerService().getStatement().executeQuery("SELECT * FROM car;");
+        List<Car> cars = new ArrayList();
 
-        allCars.forEach(car -> System.out.println(car.getBrand() + " " + car.getModel() + " - " + car.getQuantity() + " car(s), color - " + car.getColor() + ", price begins from - " + car.getPrice()));
+        while (rs.next()) {
+            cars.add(new Car (rs.getInt("id"), rs.getString("brand"), rs.getString("model"), rs.getString("color"), rs.getString("damage"), rs.getInt("price"), rs.getInt("quantity")));
```
Comment (dkozinets) [inline src/main/java/com/github/orest/car_shop/service/CarService.java]: it would look better with setters

**Pull request merged**